docs/docs/cli-reference/package-init.md

```diff
@@ -1,17 +1,22 @@
-# kurtosis package init
-This command initializes the current directory to be a [Kurtosis package][package] by creating a [`kurtosis.yml` file][kurtosis-yml] with the given package name.
+---
+title: package init
+sidebar_label: package init
+slug: /package-init
+---
+
+The `package init` command converts the current directory into a [Kurtosis package][package] by generating a new [`kurtosis.yml`][kurtosis-yml] file using the given package name.
 
 ```
 Usage:
-  kurtosis package init [flags] package_name
+  kurtosis package init [flags] $PACKAGE_NAME
 ```
 
-The `package_name` argument is the [locator][locators]to the package, in the format `github.com/USER/REPO`.
+The mandatory `#PACKAGE_NAME` argument is the [locator][locators] to the package, in the format `github.com/USER/REPO`.
 
 This command accepts the following flags:
 - `--main`: indicates that the created package is an [executable package][executable-package], and generates a `main.star` if one does not already exist. If a `main.star` already exists, does nothing.
```

Comment:
@leeederek this flag was removed because now we create the 'main' file by default